.github/workflows/ci.yml

```
@@ -107,6 +122,25 @@ jobs:
+      - name: Get version information
+        id: get_version_info
+        run: |
+          if [[ "${{ github.ref }}" == refs/tags/* ]]; then
+            # We're running on a tag push
+            VERSION_TAG="${GITHUB_REF#refs/tags/}"
+            echo "Running on tag: $VERSION_TAG"
+          else
+            # We're running after auto-tag created new tags
+            VERSION_TAG="${{ steps.get_tag.outputs.CURRENT_TAG }}"
+            echo "Using auto-created tag: $VERSION_TAG"
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)